JS/validation.js

```diff
@@ -0,0 +1,32 @@
+const form = document.querySelector('.contact-form');
+const email = document.getElementById('email');
+const emailError = document.querySelector('#email + .error');
+
+// const emailRegx = /^([a-z\d-]+)@([a-z\d-]+)\.([a-z]{2,8})(\.[a-z]{2,8})?$/;
+
```

Comment:
- [ ] Dear @mertkantaroglu, Commented codes are against a good practice for clean code as this will make your code messy. Pleas remove commented code and remember you can always go back with the commit history to see a code you have used once.